dts/riscv/gigadevice/gd32vf103.dtsi

```diff
@@ -183,6 +183,21 @@
 			status = "okay";
 		};
 
+		fwdgt: watchdog@40003000 {
+			compatible = "gd,gd32-fwdgt";
+			reg = <0x40003000 0x400>;
+			status = "disabled";
+		};
+
+		wwdgt: watchdog@40002c00 {
+			compatible = "gd,gd32-wwdgt";
+			reg = <0x40002C00 0x400>;
+			rcu-periph-clock = <0x70b>;
+			interrupt-parent = <&eclic>;
```

Comment:
redundant, parent is set at `soc` level now

Reply:
Removed this line.